src/commands.rs

```diff
@@ -320,6 +320,14 @@ pub struct ElectrumOpts {
         default_value = "10"
     )]
     pub stop_gap: usize,
+
+    /// Enable domain validation when connecting to Electrum servers.
```

Comment:
Need to make it an Option<bool> and set the default in the code, see below.

```suggestion
    /// Enable domain validation when connecting to Electrum servers  [default: true]
```

Reply:
Done.